src/brand.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Dart Brand Guidelines
+---
+
+The "Dart" name and logo are trademarks owned by Google.
+These Brand Guidelines describe the appropriate uses of the Dart
+trademarks by members of the developer community who have obtained our
+consent to use the trademarks pursuant to the [Dart Terms of Service](/terms).
+These guidelines will ensure that the Dart trademarks are used in a
+manner that promotes Google's mission to provide a free and open source
```

Comment:
Seems flutter-specific.

Reply:
The other file drops the " for crafting...record time"

Reply:
Done